src/main/java/seedu/linkedout/logic/parser/SearchCommandParser.java

```diff
@@ -15,18 +20,62 @@
     /**
      * Parses the given {@code String} of arguments in the context of the SearchCommand
      * and returns a SearchCommand object for execution.
+     *
      * @throws ParseException if the user input does not conform the expected format
      */
     public SearchCommand parse(String args) throws ParseException {
-        String trimmedArgs = args.trim();
-        if (trimmedArgs.isEmpty()) {
+
+        ArgumentMultimap argMultimap = ArgumentTokenizer.tokenize(args, PREFIX_NAME, PREFIX_JOB);
+
+        boolean noPrefixesPresent = !anyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_JOB);
```

Comment:
Apologies for requesting another change but would be good to rename in accordance to coding standards:
(Same for the other booleans used)

![image](https://user-images.githubusercontent.com/63341400/159159051-e48b5094-6c11-4f52-a16a-76c70e345540.png)


```suggestion
        boolean hasNoPrefixesPresent = !anyPrefixesPresent(argMultimap, PREFIX_NAME, PREFIX_JOB);
```